package/libs/openssl/Config.in

```diff
@@ -1,68 +1,326 @@
 if PACKAGE_libopenssl
 
+choice
+	prompt "OpenSSL Version"
+	default OPENSSL_USE_VERSION_1_1_0
+	help
+		Select the version of OpenSSL you wish to use.
+
+	config OPENSSL_USE_VERSION_1_0_2
+		bool "OpenSSL v. 1.0.2 (old LTS) series"
+		help
+		   1.0.2 is a Long Term Support (LTS) series, supported by
+		   upstream until 31st December 2009.
```

Comment:
OpenSSL 1.0.2 is supported until 31st December 2019.

Reply:
Thanks for pointing it out.